protocols/luci-proto-ipip/Makefile

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (C) 2008-2016 The LuCI Team <luci@lists.subsignal.org>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=Support for IPIP tunnels (IPv4-in-IPv4 RFC2003)
+LUCI_DEPENDS:=
+
```

Comment:
you should add:
LUCI_PKGARCH:=all

PKG_NAME:=.....
PKG_VERSION:=....
PKG_RELEASE=.....
PKG_MAINTAINER:=.....
PKG_LICENSE:=Apache-2.0